GymApp/GymAppApi/Models/CouchVisitor.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace GymAppApi.Models
+{
+    public class CouchVisitor
```

Comment:
Maybe better to rename this model to Events and maybe better to discuss data structure with your mentor